src/main/java/org/intellij/sdk/codesync/Utils.java

```diff
@@ -214,8 +216,6 @@ public static void FileCreateHandler(String filePath, String repoPath) {
         File f_originals = new File(destOriginals);
         File f_shadow = new File(destShadow);
 
-        if (f_originals.exists()) { return; }
```

Comment:
I think this was here to skip duplicate event. e.g. If new file is handled by brew daemon, it should be ignore by other plugins. was this causing trouble in some other use case?

Reply:
Yes, but it was causing issue in case the following case.

> Someone deletes a files, then he realises that he needed that file and presses  **CTRL + Z** in this case we need to update the file in `.originals` otherwise undo behaves very strange.